scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/internal/semanticdb/ConfigOps.scala

```diff
@@ -94,6 +96,18 @@ object MemberMode {
   case object None extends MemberMode
 }
 
+sealed abstract class OverrideMode {
+  import OverrideMode._
+  def isDef: Boolean = this == Def
+}
+object OverrideMode {
+  def unapply(arg: String): Option[OverrideMode] =
+    all.find(_.toString.equalsIgnoreCase(arg))
+  def all = List(Def, None)
+  case object Def extends OverrideMode
```

Comment:
What does `Def` mean here? I suggest we use `All` to be consistent with other modes.